shuup/admin/modules/orders/static_src/create/actions/index.js

```diff
@@ -66,6 +66,7 @@ export const retrieveProductData = function ({id, forLine, quantity}) {
             dispatch(receiveProductData({id, data}));
             if (forLine) {
                 dispatch(updateLineFromProduct({id: forLine, product: data}));
+                dispatch(setLineProperty(forLine, "product", data.product));
```

Comment:
I think since it's nested it doesn't set it. Currently this doesn't matter since product is set by response from browse popup window, but in the case of product select we need to get it from Json. I can double check this and make sure it's needed, but I'm pretty sure it didn't set it right otherwise.